### ISSUE TITLE
NuGet pack CPU spinning when creating a meta-package

### ISSUE THREAD

**Issue by cartbramer** [NuGet/Home#6715]
Created a simple .nuspec file: 

```
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
    <metadata>
        <id>Test</id>
        <version>0.1.0</version>
        <description>Test library</description>
        <authors>MyName</authors>
	  <dependencies>
	    <dependency id="DotSpatial.Serialization" version="1.9.0"/>
	    <dependency id="DotSpatial.Topology" version="1.9.0"/>
	    <dependency id="DotSpatial.Mono" version="1.9.0"/>
	    <dependency id="DotSpatial.Projections" version="1.9.0"/>
	    <dependency id="DotSpatial.Data" version="1.9.0"/>
	  </dependencies>
	</metadata>
</package>
```

Ran the following command: 

```
C:\Users\b.cramer\Documents\repos>nuget pack test.nuspec -Verbosity detailed
NuGet Version: 4.5.1.4879
Attempting to build package from 'test.nuspec'.
```

Command doesn't finish. The NuGet Command Line process is 'calculating' at 33% all the time. 

Turns out that if I add a "files" section, it does finish successfully (even when no "file" tags are present within). Although this is a simple workaround, I believe that this case should be handled more gracefully.

**Comment by rohit21agrawal** [comment]
@cartbramer this is likely dependent on where the nuspec file is present on disk. When the ``` <files> ``` tag is not present, it uses directory based convention, and it sounds like it's trying to pack A LOT OF FILES from the directory.

**Comment by MagicAndre1981** [comment]
use WPRUI (Windows Performance Recorder UI, select CPU and click on start, run nuget pack capture 1 minute of cpu usage, click on save). Now open the ETL with WPA to analyze the cpu usage:

https://channel9.msdn.com/Shows/Defrag-Tools/Defrag-Tools-42-WPT-CPU-Analysis

**Comment by MagicAndre1981** [comment]
> [@cartbramer](https://github.com/cartbramer) this is likely dependent on where the nuspec file is present on disk. When the `<files>` tag is not present, it uses directory based convention, and it sounds like it's trying to pack A LOT OF FILES from the directory.

yes, when there are subdirectories, nuget is adding all files and this triggers Windows Defender to scan files and this causes the high CPU usage.

So run this command in an empty folder without any subfolders or files, here it works very fast.
